core/src/main/scala/org/bitcoins/core/api/chain/ChainApi.scala

```diff
@@ -83,7 +83,7 @@ trait ChainApi extends ChainQueryApi {
     * Generates a filter header range in form of (startHeight, stopHash) by the given stop hash.
     */
   def nextFilterHeaderBatchRange(
-      prevStopHash: DoubleSha256DigestBE,
+      startHeight: Int,
```

Comment:
In the [actual message](https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki#getcfilters) it is `(startHeight, stopHash)` so this is just an optimizaition.

This should be reorg safe because the stopHash can only have one ancestor at that height